src/Core/Checkout/Cart/Address/Error/AddressValidationError.php

```diff
@@ -3,18 +3,30 @@
 namespace Shopware\Core\Checkout\Cart\Address\Error;
 
 use Shopware\Core\Checkout\Cart\Error\Error;
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 use Symfony\Component\Validator\ConstraintViolationList;
 
 #[Package('checkout')]
-class AddressValidationError extends Error
+class AddressValidationError extends Error implements AddressErrorInterface
 {
     private const KEY = 'address-invalid';
 
+    /**
+     * @deprecated tag:v6.8.0 - reason:parameter-type-change - $addressId will be required
```

Comment:
means, only the default `null` will be removed, right? should maybe stated explicitly. not that someone in the future removes the null part completely :grin:

Reply:
Nope, I want the `?` to be removed. I try to make it more clear.